integrations/aws/utils/resources.py

```diff
@@ -222,9 +232,18 @@ async def resync_cloudcontrol(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
                         e.response.get("Error", {}).get("Code")
                         == "AccessDeniedException"
                     ):
+                        if is_global:
+                            logger.warning(
+                                f"Access denied in global resource {kind} in region {region}. Querying again in all regions"
+                            )
+                            async for batch in resync_cloudcontrol(
+                                kind, is_global=False
```

Comment:
if I dont have access to region x which is the default, but we do have access to region y, z will we query both y and z?

Reply:
Yes, until the query will return some data or we run on all regions